### DISCUSSION TITLE
PAG Viewer中的Image File Rule里的时间变速（TimeRemap）功能失效，无法预览。

### DISCUSSION THREAD

**Discussion by mosaictian** [Tencent/libpag#2357] [Q&A] [answered]
<img width="282" alt="截屏2024-07-02 17 45 59" src="https://github.com/Tencent/libpag/assets/125628431/0977071e-836e-43fe-9a69-604341f12c55">
在AE中添加了TimeRemap的效果，使用了曲线变速。
但是在输出的文件中校验变速效果时，发现一旦导入视频，则Pag Viewer就会崩溃。

**Comment by domchen** [comment, marked as answer]
目前是支持不了曲线变速的哈。

**Reply by mianma12138** [reply]
管理员，卡顿的帖子我看了几个，为啥都不回复啊，是没法解决吗

**Reply by domchen** [reply]
> 管理员，卡顿的帖子我看了几个，为啥都不回复啊，是没法解决吗

暂时没回复的一般都是描述的信息不全，只说一个表现没有提供任何有参考意义的信息的话，都难以定位问题。人力有限，都是优先回复比较详细明确的提问。

**Reply by mianma12138** [reply]
https://github.com/Tencent/libpag/discussions/2419
